lib/setupmanagers/qemuhck/qemu_machine.rb

```diff
@@ -420,23 +420,23 @@ def run_vm
     end
 
     def add_hostfwd
-      [@monitor_port, @vnc_port].each do
+      [@monitor_port, @vnc_port].each do |port|
         @hostfwds << @options['slirp'].run({
                                              'execute' => 'add_hostfwd',
                                              'arguments' => {
                                                'proto' => 'tcp',
-                                               'host_port' => _1,
-                                               'guest_port' => _1
+                                               'host_port' => port,
+                                               'guest_port' => port
                                              }
                                            })
       end
     end
 
     def remove_hostfwd
-      @hostfwds.each do
+      @hostfwds.each do |fd|
```

Comment:
`fd` sounds like a file descriptor, but it is not.

Reply:
Fixed in https://github.com/HCK-CI/AutoHCK/pull/164